scrapy/templates/project/module/settings.py.tmpl

```diff
@@ -2,16 +2,67 @@
 
 # Scrapy settings for $project_name project
 #
-# For simplicity, this file contains only the most important settings by
-# default. All the other settings are documented here:
+# For simplicity, this file contains only settings considered important or
+# commonly used. You can find more settings consulting the documentation:
 #
 #     http://doc.scrapy.org/en/latest/topics/settings.html
-#
+#     http://scrapy.readthedocs.org/en/latest/topics/downloader-middleware.html
+#     http://scrapy.readthedocs.org/en/latest/topics/spider-middleware.html
 
 BOT_NAME = '$project_name'
 
 SPIDER_MODULES = ['$project_name.spiders']
 NEWSPIDER_MODULE = '$project_name.spiders'
 
+
 # Crawl responsibly by identifying yourself (and your website) on the user-agent
 #USER_AGENT = '$project_name (+http://www.yourdomain.com)'
+
+
+# Configure a delay for requests for the same website
```

Comment:
saw it, but doesn't worth to mention?
`Configure a delay for requests (affected by Autothrottle)`

`for the same website` depends on the concurrents requests you choose, default is website.